bigip/resource_bigip_ltm_monitor.go

```diff
@@ -113,6 +113,11 @@ func resourceBigipLtmMonitor() *schema.Resource {
 				Default:     "*:*",
 				Description: "Alias for the destination",
 			},
+			"compatibility": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Specifies, when enabled, that the SSL options setting (in OpenSSL) is set to ALL. The default value is enabled.",
```

Comment:
Might be a good idea to add a ValidateFunc to check that the value provided is either `enabled` or `disabled`.